package.json

```diff
@@ -26,17 +26,24 @@
   "scripts": {
     "clean": "rimraf ./dist src/content/**/_*.md src/**/_*.json",
     "start": "cross-env NODE_ENV=development webpack-dev-server --config webpack.dev.js --env.dev",
+    "update-repos": "node src/utilities/fetch-package-repos.js",
     "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "build-test": "npm run build && http-server build/",
-    "fetch": "sh src/scripts/fetch.sh",
+    "fetch": "run-p fetch:*",
+    "fetch:readmes": "node src/utilities/fetch-package-readmes.js",
+    "fetch:supporters": "node src/utilities/fetch-supporters.js",
+    "fetch:starter-kits": "node src/utilities/fetch-starter-kits.js",
     "prebuild": "npm run clean",
-    "build": "npm run fetch && npm run content && cross-env NODE_ENV=production webpack --config webpack.prod.js && npm run sitemap && echo webpack.js.org > dist/CNAME",
+    "build": "run-s fetch content && cross-env NODE_ENV=production webpack --config webpack.prod.js",
+    "postbuild": "npm run sitemap",
     "test": "npm run lint",
     "lint": "run-s lint:*",
-    "lint:js": "eslint src --ext .js,.jsx,.md",
+    "lint:js": "eslint src --ext .js,.jsx,.md --cache true --cache-location .cache/.eslintcache",
     "lint:markdown": "markdownlint --config ./.markdownlint.json *.md ./src/content/**/*.md --ignore './src/content/**/_*.md'",
     "lint:social": "alex . -q",
     "lint:prose": "cp .proselintrc ~/ && proselint src/content",
+    "lint:links": "hyperlink -r dist/index.html --canonicalroot https://webpack.js.org/ -i | tee internal-links.tap | tap-spot",
```

Comment:
What is `tee internal-links.tap`?

Reply:
Tee writes a copy of stdout to a file while paying through the stdout pipe. It let's us keep a copy of the raw tap output while seeing the nice formatted report in the console. It's just in case we need to go back and look at the output in a different manner, for example if the reporter has a bug, or we forgot to copy/paste the console output

Reply:
It is a dependency or something available in linux?

Reply:
Oh yeah, linux program, cool! TIL